### DISCUSSION TITLE
Question - Move pan area programmatically

### DISCUSSION THREAD

**Discussion by peacepostman** [chartjs/chartjs-plugin-zoom#702] [Q&A]
Hello,

I would like to move the pan from a zoomed graph by dragging an external component. 

It seems feasible with the help of the `pan` function nevertheless i'm stuck with applying the proper x value relative to the total graph size. Is it possible to retrieve the current pan value and the total 'panable' area size in order to apply the proper x value ?

Thanks,

**Comment by kurkle** [comment]
You should be able to use the scales of the chart to get proper values.
